src/seedu/addressbook/commands/DeleteCommand.java

```diff
@@ -11,27 +14,36 @@
 public class DeleteCommand extends Command {
 
     public static final String COMMAND_WORD = "delete";
-
+    
```

Comment:
Extra blank line. Use blank lines sparingly. It's good to have blank lines to separate logical code blocks, but too many blank lines will increase the height of the code, which increases the amount of scrolling.